### PR TITLE
added scale for hovering over color options

### PR DESCRIPTION
**Pull request by tron975**
I basically just made the color picker options get bigger when you hover over them

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -174,7 +174,7 @@ <h2><button class= "newFeature">NEW</button> Font Family</h2>
 <img src="img/pencil.png" class="headerButton" style="right:5px; top: 80px;" onclick="toggle()">
 <div id="draw" style="width: 100%; height: 100%; display: none;">
     <canvas style="position: absolute; top: 0; left: 0; z-index: 5;" cursor: pointer; id="can"></canvas>
-    <div class="chooseColor" style="border: 2px solid white; top:170px;right:18px;background:green; width: 30px; height: 30px; border-color: white;" id="green" onclick="color(this)"></div>
+    <div onclick = document.querySelectorAll("chooseColor").style.backgroundColor = "red" class="chooseColor" style="border: 2px solid white; top:170px;right:18px;background:green; width: 30px; height: 30px; border-color: white;" id="green" onclick="color(this)"></div>
```
Comment (Polydynamical) [inline index.html:177]: Why are you changing the background color to red?

File: js/script.js [hunk 2]
```diff
@@ -38,6 +38,10 @@ function saveFont() {
     textFont();
 }
 
+function checkMark() {
+    onClick = document.getElementsByClassName("chooseColor").style.color = "red"
```
Comment (Polydynamical) [inline js/script.js:42]: Same here, why is the check mark red? I was thinking a black check mark...

File: css/style.css [hunk 3]
```diff
@@ -137,8 +137,12 @@ p {
     width: 15px;
     height: 15px;
     cursor: pointer;
+    transition: 0.4s;
```
Comment (Polydynamical) [inline css/style.css:140]: This could be a bit faster, maybe 0.3 or 0.25

**Review by Polydynamical — CHANGES_REQUESTED** [review]

**Comment by tron975** [comment]
this was a pull request test/mistake

**Pull request closed without merging**